appium/webdriver/appium_connection.py

```diff
@@ -25,21 +25,38 @@
 
 
 class AppiumConnection(RemoteConnection):
-    def _get_connection_manager(self) -> Union[urllib3.PoolManager, urllib3.ProxyManager]:
-        # https://github.com/SeleniumHQ/selenium/blob/0e0194b0e52a34e7df4b841f1ed74506beea5c3e/py/selenium/webdriver/remote/remote_connection.py#L134
-        pool_manager_init_args = {'timeout': self._timeout}
+    def __init__(
+        self, remote_server_addr: Union[str, bytes], keep_alive: bool = False, ignore_proxy: Optional[bool] = False
+    ):
+        self._pool_manager_init_args: Dict[str, Any] = {}
```

Comment:
not sure if we need to set the default value here if we anyway redefine it in set_init_args_for_pool_manager